5_databases/week1/homework.sql

```diff
@@ -0,0 +1,52 @@
+USE hyf_lesson1;
+
+-- 1 Find out how many tasks are in the task table
+
+SELECT COUNT (title) FROM task 
+
+-- 2 Find out how many tasks in the task table do not have a valid due date
+
+SELECT * FROM task WHERE due_date IS NULL 
+
+-- 3 Find all the tasks that are marked as done
+
+SELECT * FROM task WHERE status_id = 3 
+
+-- 4 Find all the tasks that are not marked as done
+
+SELECT * FROM task WHERE status_id != 3 
+
+-- 5 Get all the tasks, sorted with the most recently created first
+
+SELECT * FROM task ORDER BY created 
```

Comment:
You just need to add DESC for descending to get most recent:

SELECT * 
FROM task 
ORDER BY created DESC;